woof-code/rootfs-skeleton/usr/sbin/quicksetup

```diff
@@ -1173,6 +1187,9 @@ if [ "$SET_KEYBOARD" ];then
  fi
  if [ -f ~/.Xwaylandrc ]; then
   sed -i "s%^XKB_DEFAULT_LAYOUT=.*%XKB_DEFAULT_LAYOUT=$XKB_LAYOUT%" ~/.Xwaylandrc
+  if [ -f "$XDG_CONFIG_HOME/labwc/environment" ]; then
```

Comment:
I think this `if` should be outside of the previous one - we want it to be possible to have labwc without Cage and Xwayland